README.md

```diff
@@ -1 +1,160 @@
-# webpackSetup
\ No newline at end of file
+
+          To do List
+
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>To do List </b></h3>
+</div>
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 to do List <a name="about-project"></a>
+
+"To do list" is a minimalist web application that uses HTML, CSS, webpack, bundlers and linters to let a user create a to do list. Usr would be able add and remove to do list items dynamically and the list items will be generated dynamically using javacsript (ES6).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Java Script</li>
+  </ul>
+</details
+<!-- Features -->
+### Key Features <a name="key-features"></a>
+<ul>
+<li> User is able to add a list item for todo list.</li>
+<li> User is able to 'check' in checkbox for completed list items .</li>
+<li> The webapp enables user to see all the to do items to be done.</li>
+<li> Contact form for users to get in touch with the developer.</li>
+<li> webapp uses minimalist web design.</li>
+<li> UX is manifested by simplicity and easy to use interaction</li>
+</ul>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+```sh
+coming soon
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Git: Download and install Git for your platform.
+- Node.js: Download and install Node.js for your platform
+- webpack and liver server
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:umarahgul/webpackSetup.git
+```
+
+### Install
+
+Install linters for this project with:
+Install webpack
+
+```sh
+  cd my-project
+  npm install
+```
+### Deployment
+  
+  ```sh
+   https://umarahgul.github.io/Portfolio/  
+```
+ 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Umarahgul**
+
+GitHub: [@umarahgul](https://github.com/umarahgul)
+LinkedIn: [umarahgul](https://www.linkedein.com/umarahgul)
```

Comment:
- [ ] As stated by the previous reviewer, the link you have here for your LinkedIn page doe not take us to the LinkedIn page. I would kindly suggest you add the correct link. Thank you
Also, since you don't have the project deployed yet, kindly remove the live demo section as this will make your Readme look professional